com.unity.render-pipelines.high-definition/Documentation~/Upgrading-To-HDRP.md

```diff
@@ -53,6 +53,8 @@ To upgrade the Materials in your Scene to HDRP-compatible Materials, either:
 
 * **Upgrade Selected Materials to High Definition Materials**: Converts every compatible Material currently selected in the Project window to an HDRP Material.
 
+* **Upgrade Scene Terrains to High Definition Terrains**: Replaces the built-in default standard terrain Material in every terrain in the scene with HDRP default Terrain Material.
```

Comment:
Replaces the standard built-in terrain Material for every [Terrain](https://docs.unity3d.com/Manual/script-Terrain.html) in the scene with the HDRP default Terrain Material.

Reply:
Modified ! Thanks 👍